src/components/skinmanager.js

```diff
@@ -219,7 +219,7 @@ define(['apphost', 'userSettings', 'browser', 'events', 'pluginManager', 'backdr
             selectedTheme = defaultTheme;
         }
 
-        var embyWebComponentsBowerPath = 'bower_components/emby-webcomponents';
```

Comment:
place a TODO to fix variable name?

Reply:
made a commit renaming all embyWebComponentsBowerPath to componentsPath. This fixes the TODO?

Reply:
yeah, thx!